Specs/Scene/DiscardEmptyTileImagePolicySpec.js

```diff
@@ -0,0 +1,54 @@
+defineSuite([
+        'Scene/DiscardEmptyTileImagePolicy',
+        'Core/Resource',
+        'Specs/pollToPromise',
+        'ThirdParty/when'
+    ], function(
+        DiscardEmptyTileImagePolicy,
+        Resource,
+        pollToPromise,
+        when) {
+    'use strict';
+
+    afterEach(function() {
+        Resource._Implementations.createImage = Resource._DefaultImplementations.createImage;
+        Resource._Implementations.loadWithXhr = Resource._DefaultImplementations.loadWithXhr;
+    });
+
+    describe('shouldDiscardImage', function() {
+        fit('does not discard a non-empty image', function() {
```

Comment:
`fit` snuck into the committed code here, which disables all other tests.  (this sort of thing is why `fit` should just not exist in Jasmine, in my opinion)

Reply:
`fit` and `fdescribe` are really helpful if you just need to debug one new test.  However, in some of our other repos we added a custom eslint rule to catch it so it would break CI if you committed it.  Maybe we should add that to this repo too

Reply:
The spec filtering can run single tests or suites without needing to edit the source, e.g.:

http://localhost:8080/Specs/SpecRunner.html?spec=Core%2FappendForwardSlash%20Appends%20to%20a%20url

Reply:
That doesn't work if you need to run it from the command line to reproduce an error that only happens in travis

Reply:
our custom web client is honestly crazy

Reply:
Doesn't our `eslint` configuraton flag fit/fdescribe?  It does everywhere else outside of Cesium.

Reply:
No, I'm pretty sure it's just in the `eslintrc.json` for specific projects

Reply:
Play. We should definitely fix that then, I'll open a PR soon-ish

Reply:
See #7809

Reply:
How embarrassing, good catch. 😳